host/src/processligand.cpp

```diff
@@ -23,6 +23,9 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
 */
 
 
+// Use AD4 minimum and maximum bond distances instead of the Frankenstein we had in AD-GPU
```

Comment:
Minor thing: I would suggest a more descriptive comment on what this `#define` does, like the description of this PR https://github.com/ccsb-scripps/AutoDock-GPU/pull/85#issue-436120774